### PR TITLE
Remove deprecated android5, win8 and win10 themes

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

**Review by BingoRUS — DISMISSED** [review]
👍

**Comment by alexanderPolosatov** [comment]
To this PR necessary add removing deprecated 'mobile' themes too. 
Do not merge this PR before it.
//CC @San4es

**Comment by KonyukhovSergey** [comment]
@BingoRUS please check this https://github.com/DevExpress/DevExtreme/pull/7497/commits/1ee351f06502f8cd162d655062728475fa89f861 commit

File: js/ui/toolbar.js [hunk 1]
```diff
@@ -169,30 +169,6 @@ var Toolbar = ToolbarBase.inherit({
                 options: {
                     submenuType: "actionSheet"
                 }
-            },
-            {
-                device: function() {
```
Comment (KonyukhovSergey) [inline js/ui/toolbar.js:174]: @jsdmitry please review this part. Maybe the 'dropDownMenu', 'listBottom', 'listTop' submenuTypes now are useless.

Comment (San4es) [reply js/ui/toolbar.js:174]: @jsdmitry @jauseg 
Guys, is this issue resolved?

File: js/ui/themes.js [hunk 2]
```diff
@@ -334,28 +320,6 @@ function isGeneric(themeName) {
...
 
-function isWin8(themeName) {
-    return isTheme("win8", themeName);
-}
-
-function isWin10(themeName) {
-    return isTheme("win10", themeName);
-}
-
-function checkThemeDeprecation() {
```
Comment (San4es) [inline js/ui/themes.js]: @BingoRUS @jauseg 
maybe we need to save this function to use it for the ios7 theme deprecation?

File: js/ui/toolbar.js (hunk 1 above)
Comment (KonyukhovSergey) [reply js/ui/toolbar.js:174]: Yes, we have reviewed and resolved these submenuTypes. The 'dropDownMenu' and 'actionSheet' use in the code.

File: testing/tests/DevExpress.ui.widgets/toolbar.tests.js [hunk 3]
```diff
@@ -242,105 +238,6 @@ QUnit.module("toolbar with menu", {
     }
 });
 
-QUnit.test("render menu items", function(assert) {
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets/toolbar.tests.js:245]: Breaking change?

**Review by San4es — CHANGES_REQUESTED** [review]

File: js/ui/toolbar.js (hunk 1 above)
Comment (San4es) [reply js/ui/toolbar.js:174]: BC?

**Review by jsdmitry — DISMISSED** [review]

File: js/ui/toolbar.js (hunk 1 above)
Comment (KonyukhovSergey) [reply js/ui/toolbar.js:174]: I think it does not BC. The submenuType is a private type. The value of this type depends on the theme. But these themes removed.

File: testing/tests/DevExpress.ui.widgets/toolbar.tests.js (hunk 3 above)
Comment (KonyukhovSergey) [reply testing/tests/DevExpress.ui.widgets/toolbar.tests.js:245]: I think it doesn't BC by the same reason https://github.com/DevExpress/DevExtreme/pull/7497#discussion_r274313416

File: js/ui/themes.js (hunk 2 above)
Comment (KonyukhovSergey) [reply js/ui/themes.js]: I have reverted this functional with the https://github.com/DevExpress/DevExtreme/pull/7497/commits/9944adcab4bb71366bda88a190728866e3a0c1b1 commit

**Review by San4es — DISMISSED** [review]
LGTM. Let's merge it before 19.1 beta!

**Review by BingoRUS — DISMISSED** [review]

**Review by KonyukhovSergey — APPROVED** [review]

**Pull request merged**